source/jobs/_contact_us.erb

```diff
@@ -5,7 +5,7 @@
     </h2>
 
     <p class="text-tile__paragraph--with-spacing">
-      Steve handles recruitment for Unboxed and would love to hear from you.
+      Steve is our people person. If this role sounds of interest to you, he would love to hear from you.
```

Comment:
I don't think this sentence quite works when this component is shown on the 'join us' page in the case when we have more than one role available. Is there a way we can either make this sentence generic for both pages or perhaps pass in a count of roles when we make the call to `<%= partial 'jobs/contact_us' %>` and display a different wording accordingly?

Reply:
Good idea, I have added a check for the number of roles in the commit now